test/spec/transmissions.spec.js

```diff
@@ -1,27 +1,29 @@
 var chai = require('chai')
   , expect = chai.expect
   , sinon = require('sinon')
-  , sinonChai = require('sinon-chai')
   /* global -Promise */
   , Promise = require('../../lib/Promise');
 
-chai.use(sinonChai);
+require('sinon-as-promised');
+
+chai.use(require('sinon-chai'));
+chai.use(require('chai-as-promised'));
```

Comment:
it looks like all tests are using `done` style test. so why you need `chai-as-promised`?